opcua/common/instantiate.py

```diff
@@ -86,7 +89,8 @@ def _instantiate_node(server, parentid, rdesc, nodeid, bname, dname=None, recurs
                         nodeids = _instantiate_node(server, res.AddedNodeId, c_rdesc, nodeid=ua.NodeId(identifier=inst_nodeid, namespaceidx=res.AddedNodeId.NamespaceIndex), bname=c_rdesc.BrowseName)
                     else:
                         nodeids = _instantiate_node(server, res.AddedNodeId, c_rdesc, nodeid=ua.NodeId(namespaceidx=res.AddedNodeId.NamespaceIndex), bname=c_rdesc.BrowseName)
-                    added_nodes.extend(nodeids)
+                    if nodeids is not None:
+                        added_nodes.extend(nodeids)
```

Comment:
when would this happen? I think that if _instantiate does not return a node then something wrong has happend and we should raise an exception (If instantiate method has not done it already)